infrastructure/aws/production/ecs-task-definition-task-processor.json

```diff
@@ -123,9 +123,17 @@
                     "name": "PIPEDRIVE_LEAD_LABEL_EXISTING_CUSTOMER_ID",
                     "value": "d828a3a0-bf34-11ed-b08d-b12d3497f5c4"
                 },
+                {
+                    "name": "TASK_DELETE_RUN_TIME",
+                    "value": "07:00"
```

Comment:
I've added this now so that, when we do want to run the task in production it is run at our quietest time.